governance/votes/Executive vote - July 11, 2024.md

```diff
@@ -0,0 +1,125 @@
+
+| Delegate    | Amount (MKR) | Address                                                                                         |
+|-------------|--------------|-------------------------------------------------------------------------------------------------|
+| BLUE        | 41.67       | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
+| JuliaChang  | 41.67        | [0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7](https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7) |
+| Byteron     | 38.98         | [0xc2982e72D060cab2387Dba96b846acb8c96EfF66](https://etherscan.io/address/0xc2982e72D060cab2387Dba96b846acb8c96EfF66) |
+| Cloaky      | 20.40 (requested)        | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) |
+| PBG         | 16.58        |[0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) |
+| BONAPUBLICA | 13.89        | [0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3) 
+| Rocky       | 13.89        |[0xC31637BDA32a0811E39456A59022D2C386cb2C85](https://etherscan.io/address/0xc31637bda32a0811e39456a59022d2c386cb2c85) |
+| vigilant    | 12.55        | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) |
+| Ennoia (Cloaky)  | 4.10  (requested)   | [0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b](https://etherscan.io/address/0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b) |
+| Kohla (Cloaky)  | 4.10  (requested)   | [0xA9D43465B43ab95050140668c87A2106C73CA811](https://etherscan.io/address/0xA9D43465B43ab95050140668c87A2106C73CA811) |
+| WBC         | 1.34        |[0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) |
+
+### Spark Proxy Spell
+
+**Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594/2), [Governance Poll 1120](  https://vote.makerdao.com/polling/QmTBsxR5)  
+**Proposal**: [Stability Scope Parameter Proposal](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594), [[Jun 27, 2024] Proposed Changes to Spark for Upcoming Spell](https://forum.makerdao.com/t/jun-27-2024-proposed-changes-to-spark-for-upcoming-spell/24552)
+
+If this executive proposal passes a Spark Proxy spell will be triggered at [0x91824fa4fd51E8440a122ffDd49C701F5C56D58e](https://etherscan.io/address/0x91824fa4fd51E8440a122ffDd49C701F5C56D58e). The expected inclusions are as follows:
+
+#### [Mainnet] Spark Effective DAI Borrow Rate Reduction
+
+The Spark Effective DAI Borrow Rate will be reduced by 1 percentage point, from 9% to **8%**.
+
+#### [Mainnet] Increase Capacity of weETH
+
+- Increase weETH Supply Cap Max by 150,000 weETH, from 50,000 weETH to **200,000 weETH**
```

Comment:
```suggestion
- weETH Supply Cap Max will be increased by 150,000 weETH, from 50,000 weETH to **200,000 weETH**
```
for the sake of consistency in style and form

Reply:
This might be my error from how I inputted the items into the Exec sheet. To keep this as consistent as possible with the sheet language, I'm going to change the DAI borrow rate item. That way the form is the same across these three, and spell sheet is still largely word for word with what is being communicated to voters. 

Do we have a section about this for the style guide? Because while the sheet should say "Increase to X" I do like the "will be increased" format we've been using for the spell copy. Maybe the convention should be that the sheet describes the change and the format for the copy becomes "[Parameter] will be [increased/reduced]..."

Reply:
It was taken verbatim from the poll: https://github.com/makerdao/community/blob/master/governance/polls/SparkLend%20Ethereum%20-%20Increase%20Capacity%20of%20weETH%20-%20July%201%2C%202024.md#review